doc/usage.md

```diff
@@ -195,7 +194,7 @@ CodeCompass_webserver -w <workdir> -p <port>
   connection string the user can give the location of a running database
   system. In the parsing session the database name could have been provided.
   This database name is written in a config file in the workspace directory, so
-  it is useless to provide it at server start.
```

Comment:
Remove the entire "Database" element from this list instead, it's no longer present or used.

Reply:
That is good with me I just didn't want to remove someones work without a word.

Reply:
I believe it's actually an *error* to provide `-d` for the server binary, as that thing is no longer in the `boost::po` as a key.